pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2153,6 +2176,16 @@ def plot_model(
         # checking display_format parameter
         self.plot_model_check_display_format_(display_format=display_format)
 
+        if plot == "decomp_classical":
+            msg = (
+                "DeprecationWarning: `decomp_classical` plot type will be disabled in "
+                "a future release. Please use `decomp` instead."
+            )
+            warnings.warn(msg, DeprecationWarning)
+            print(msg)
```

Comment:
can we log instead of print? the warning will be communicated to the user regardless.

Reply:
Ok. But this would get removed in 3.00 anyway. I will try to change this in the next PR in case we decide to keep it for longer.